### PR TITLE
Eng 14347 exclude schema from usos

### PR DESCRIPTION
**Pull request by wweiss-voltdb**

### CODE REVIEW

File: src/frontend/org/voltdb/export/processors/GuestProcessor.java [hunk 1]
```diff
@@ -338,8 +338,24 @@ public void run() {
...
+                                assert(version == 1);
+                                long generation = buf.getLong();
+                                int schemaSize = buf.getInt();
+                                ExportRow previousRow = edb.getPreviousRow();
+                                if (previousRow == null || previousRow.generation != generation) {
+                                    byte[] schemadata = new byte[schemaSize];
+                                    buf.get(schemadata, 0, schemaSize);
+                                    ByteBuffer sbuf = ByteBuffer.wrap(schemadata);
+                                    sbuf.order(ByteOrder.LITTLE_ENDIAN);
+                                    edb.setPreviousRow(ExportRow.decodeBufferSchema(sbuf, schemaSize, source.getPartitionId(), generation));
```
Comment (rdykiel) [inline src/frontend/org/voltdb/export/processors/GuestProcessor.java]: Just for my understanding, 2 questions:
1- every buffer has room for schema, but it's only filled on generation change?
2- on generation change, the new shema is stored in the previous row? Why not store it in current row?

File: src/ee/storage/ExportTupleStream.h [hunk 2]
```diff
@@ -27,11 +27,20 @@ namespace voltdb {
 
 class StreamBlock;
 
+//If you change this constant here change it in Java in the StreamBlockQueue where
+//it is used to calculate the number of bytes queued
+//I am not sure if the statements on the previous 2 lines are correct. I didn't see anything in SBQ that would care
+//It just reports the size of used bytes and not the size of the allocation
+//Add a 4k page at the end for bytes beyond the 2 meg row limit due to null mask and length prefix and so on
+//Necessary for very large rows
+const int EL_BUFFER_SIZE = /* 1024; */ (2 * 1024 * 1024) + MAGIC_HEADER_SPACE_FOR_JAVA + (4096 - MAGIC_HEADER_SPACE_FOR_JAVA);
```
Comment (rdykiel) [inline src/ee/storage/ExportTupleStream.h]: + MAGIC_HEADER_SPACE_FOR_JAVA  - MAGIC_HEADER_SPACE_FOR_JAVA = 0
what's the point? Is this a way of documenting the buffer structure?

File: src/frontend/org/voltdb/export/processors/StandaloneGuestProcessor.java [hunk 3]
```diff
@@ -168,9 +168,19 @@ public void run() {
                                 final ByteBuffer buf = cont.b();
                                 buf.position(startPosition);
                                 buf.order(ByteOrder.LITTLE_ENDIAN);
-                                long generation = -1L;
+                                byte version = buf.get();
+                                assert(version == 1);
```
Comment (zeemanhe) [inline src/frontend/org/voltdb/export/processors/StandaloneGuestProcessor.java]: Will the version be changed?

**Comment by zeemanhe** [comment]
There are system test errors related to truncation on this branch.

**Pull request merged**